microsoft/testsuites/acc/cvm.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.environment import EnvironmentStatus
+from lisa.tools import Dmesg, Lsvmbus
+
+
+@TestSuiteMetadata(
+    area="ACC_CVM",
+    category="functional",
+    description="""
+    This test suite ensures configuration and devices for CVM
+    """,
+)
+class CVMSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This case verifies that lsvmbus only shows devices
+        that are allowed in a CVM guest
+
+        Steps:
+        1. Call lsvmbus
+        2. Iterate through list returned by lsvmbus to ensure all devices
+           listed are included in valid_class_ids
+        """,
+        priority=1,
+        requirement=simple_requirement(
```

Comment:
@LiliDeng I just merged Kameron's change about CVM. Please let Jose know how to test with feature enabled.

Reply:
I was able to test with Kameron's changes successfully, the only caveat is that I needed to specify image, and sku with:

-v "marketplace_image:canonical 0001-com-ubuntu-confidential-vm-focal 20_04-lts-cvm latest" 
-v "vm_size:Standard_DC16ads_v5"

Reply:
great